zpeclient_test.go

```diff
@@ -76,7 +112,72 @@ func getFakeDomain() zms.SignedDomain {
 				},
 				{
 					Name:  zms.ResourceName(domainName + ":role.delegated"),
-					Trust: "test.delegated.domain",
+					Trust: trustDomainName,
+				},
+			},
+			Services: []*zms.ServiceIdentity{},
+			Entities: []*zms.Entity{},
+		},
+		KeyId:     "colo-env-1.1",
+		Signature: "signature",
+	}
+}
+
+func getFakeTrustAthenzDomains() *v1.AthenzDomain {
```

Comment:
Could we remove the `s` at the end in the function name and also do the same for the `getFakeAthenzDomains` function as well.

Reply:
done